docs/operator-manual/webhook.md

```diff
@@ -27,11 +27,13 @@ accessible, then configuring a webhook secret is recommended to prevent a DDoS a
 In the `argocd-secret` kubernetes secret, configure one of the following keys with the Git
 provider's webhook secret configured in step 1.
 
-| Provider  | K8s Secret Key           |
-|---------- | ------------------------ |
-| GitHub    | `github.webhook.secret`  |
-| GitLab    | `gitlab.webhook.secret`  |
-| BitBucket | `bitbucket.webhook.uuid` |
+| Provider        | K8s Secret Key                   |
+|-----------------| ---------------------------------|
+| GitHub          | `webhook.github.secret`          |
+| GitLab          | `webhook.gitlab.secret`          |
+| BitBucket       | `webhook.bitbucket.uuid`         |
+| BitBucketServer | `webhook.bitbucketserver.secret` |
+| Gogs            | `webhook.gogs.secret`            |
```

Comment:
given Gogs is self-hosted, what certs do you need?

Reply:
The webhook secrets are used as "shared secret" to generate/check the payload HMAC signature to trust the sender.